Nightly build failing with org.openhab.io:org.openhab.io.multimedia.tts.freetts

The nightly builds are failiing since 4 days:
```
03:12:25.544  |  [INFO] Resolving dependencies of MavenProject: org.openhab.io:org.openhab.io.multimedia.tts.freetts:1.9.0-SNAPSHOT @ /scratch/jenkins/workspace/openHAB1-Addons/bundles/io/org.openhab.io.multimedia.tts.freetts/pom.xml
03:12:25.552  |  [INFO] {osgi.ws=gtk, osgi.os=linux, osgi.arch=x86, org.eclipse.update.install.features=true}
03:12:25.552  |  [ERROR] Cannot resolve project dependencies:
03:12:25.553  |  [ERROR]   Software being installed: org.openhab.io.multimedia.tts.freetts 1.9.0.qualifier
03:12:25.553  |  [ERROR]   Missing requirement: org.openhab.io.multimedia.tts.freetts 1.9.0.qualifier requires 'package org.openhab.io.multimedia.tts 0.0.0' but it could not be found
03:12:25.553  |  [ERROR] 
03:12:25.553  |  [ERROR] See http://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
03:12:25.556  |  [ERROR] Cannot resolve dependencies of MavenProject: org.openhab.io:org.openhab.io.multimedia.tts.freetts:1.9.0-SNAPSHOT @ /scratch/jenkins/workspace/openHAB1-Addons/bundles/io/org.openhab.io.multimedia.tts.freetts/pom.xml: See log for details -> [Help 1]
03:12:25.556  |  [ERROR] 
03:12:25.557  |  [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
03:12:25.557  |  [ERROR] Re-run Maven using the -X switch to enable full debug logging.
03:12:25.557  |  [ERROR] 
03:12:25.558  |  [ERROR] For more information about the errors and possible solutions, please read the following articles:
03:12:25.558  |  [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
```

--- a/bundles/io/pom.xml
+++ b/bundles/io/pom.xml
@@ -23,12 +23,12 @@
     <module>org.openhab.io.gcal.test</module>
     <module>org.openhab.io.gpio</module>
     <module>org.openhab.io.harmonyhub</module>
-    <module>org.openhab.io.multimedia.tts.freetts</module>
+<!--    <module>org.openhab.io.multimedia.tts.freetts</module>
     <module>org.openhab.io.multimedia.tts.googletts</module>
     <module>org.openhab.io.multimedia.tts.googletts.test</module>
     <module>org.openhab.io.multimedia.tts.macintalk</module>
     <module>org.openhab.io.multimedia.tts.marytts</module>
-    <module>org.openhab.io.multimedia.tts.speechdispatcher</module>
+    <module>org.openhab.io.multimedia.tts.speechdispatcher</module>-->
     <!--module>org.openhab.io.myopenhab</module-->
     <module>org.openhab.io.squeezeserver</module>
     <module>org.openhab.io.transport.cul</module>

--- a/features/openhab-addons/src/main/feature/feature.xml
+++ b/features/openhab-addons/src/main/feature/feature.xml
@@ -665,15 +665,6 @@
       </feature>
   -->
 
-  <!-- voice -->
-
-  <feature name="openhab-voice-marytts" description="Mary TTS" version="${project.version}">
-    <feature>openhab-runtime-base</feature>
-    <feature>openhab-runtime-compat1x</feature>
-    <bundle start-level="80">mvn:org.openhab.io/org.openhab.io.multimedia.tts.marytts/${project.version}</bundle>
-    <configfile finalname="${openhab.conf}/services/marytts.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/marytts</configfile>
-  </feature>
-
   <!-- shared features, not end user facing -->
   <feature name="openhab-io-gpio" description="GPIO IO" version="${project.version}">
     <feature>openhab-runtime-base</feature>

